pkg/toolsets/kiali/manage_istio_config.go

```diff
@@ -0,0 +1,117 @@
+package kiali
+
+import (
+	"fmt"
+
+	"github.com/google/jsonschema-go/jsonschema"
+	"k8s.io/utils/ptr"
+
+	"github.com/containers/kubernetes-mcp-server/pkg/api"
+)
+
+func initManageIstioConfig() []api.ServerTool {
+	ret := make([]api.ServerTool, 0)
+	ret = append(ret, api.ServerTool{
+		Tool: api.Tool{
+			Name:        "kiali_manage_istio_config",
+			Description: "Manages Istio configuration objects (Gateways, VirtualServices, etc.). Can list (objects and validations), get, create, patch, or delete objects",
+			InputSchema: &jsonschema.Schema{
+				Type: "object",
+				Properties: map[string]*jsonschema.Schema{
+					"action": {
+						Type:        "string",
+						Description: "Action to perform: list, get, create, patch, or delete",
+					},
+					"namespace": {
+						Type:        "string",
+						Description: "Namespace containing the Istio object",
+					},
+					"group": {
+						Type:        "string",
+						Description: "API group of the Istio object (e.g., 'networking.istio.io', 'gateway.networking.k8s.io')",
+					},
+					"version": {
+						Type:        "string",
+						Description: "API version of the Istio object (e.g., 'v1', 'v1beta1')",
+					},
+					"kind": {
+						Type:        "string",
+						Description: "Kind of the Istio object (e.g., 'DestinationRule', 'VirtualService', 'HTTPRoute', 'Gateway')",
+					},
+					"name": {
+						Type:        "string",
+						Description: "Name of the Istio object",
+					},
+					"json_data": {
+						Type:        "string",
+						Description: "JSON data to apply or create the object",
+					},
+				},
+				Required: []string{"action"},
+			},
+			Annotations: api.ToolAnnotations{
+				Title:           "Manage Istio Config: List, Get, Create, Patch, Delete",
+				ReadOnlyHint:    ptr.To(true),
+				DestructiveHint: ptr.To(false),
```

Comment:
```suggestion
				ReadOnlyHint:    ptr.To(false),
				DestructiveHint: ptr.To(true),
```

After changing the purpose of the tool, the annotations are now incorrect.